classes/CartRule.php

```diff
@@ -295,6 +295,34 @@ public static function getIdByCode($code)
     }
 
     /**
+     * Check if some cart rules exists today for the given customer
+     *
+     * @param int $idCustomer
+     *
+     * @return bool
+     */
+    public static function haveCartRuleToday($idCustomer)
+    {
+        static $haveCartRuleToday = array();
```

Comment:
@tomlev, something to say regarding that line?

Reply:
This one is not too annoying for the tests since we just have to change the idCustomer